src/hotspot/cpu/aarch64/c1_LIRGenerator_aarch64.cpp

```diff
@@ -832,9 +832,11 @@ void LIRGenerator::do_LibmIntrinsic(Intrinsic* x) {
       }
       break;
     case vmIntrinsics::_dlog:
-      // Math.log intrinsic is not implemented on AArch64 (see JDK-8210858),
```

Comment:
Drive-by comment, and since you are removing this comment:
What is the state of https://bugs.openjdk.org/browse/JDK-8210858 ?